### DISCUSSION TITLE
¿Windows o Linux?

### DISCUSSION THREAD

**Discussion by ByLobs** [Launch-X-Latam/MisionFrontEnd#83] [General]
¿Qué opinión tienen respecto a sistemas operativos base Linux? alguno que recomienden o se quedan con Windows

**Comment by Cerecero** [comment]
Yo prefiero Linux ya solo uso Windows para jugar jaja, pero creo que puedes usar los dos con dual-boot, uno recomendado, completo y de facil transicion entre Windows seria el Ubuntu para mi gusto.

**Reply by ByLobs** [reply]
Buena idea, ir probando los dos a la ves y si me convence Linux hacer el cambio de poco a poco

**Comment by NiiSV811** [comment]
La verdad, como más te sientas cómodo, yo uso Manjaro (basado en Arch) nunca he tenido problemas y me va bien, consume menos que Windows y se puede customizar mucho, si tienes la curiosidad de probar linux lo puedes hacer con un Live USB o una Máquina virtual ya si te gusta y te da curiosidad moverle a lo que haya por ahí no está mal. Si lo que quieres es algo fácil de usar y que no toque hacer nada, es preferible usar Windows, si es para Gaming también te recomendaría Windows. Es solo instalarlo y usarlo. Yo uso los 2, manjaro en una partición para estudiar, programar y así y Windows para jugar. Si es desarrollo web, cualquiera jala bien
![image](https://user-images.githubusercontent.com/88354655/193102837-6e398418-a76e-4e46-8d85-0e25a037c074.png)
Foto pa llamar la atención xD

**Reply by manuel260300** [reply]
pido link del fondo colega xd

**Reply by JosafatJimenezB** [reply]
link del fondo x2 xd

**Reply by NiiSV811** [reply]
Fuente del Wallpaper: https://twitter.com/Gznf1221/status/1369000241450557440/photo/1

**Comment by manuel260300** [comment]
Yo la verdad prefiero Linux para programar, obtengo mucho rendimiento que con windows, claro también depende de las especificaciones de tu equipo. Por ejemplo actualmente trabajo con Ubuntu Studio y me va de maravilla
![image](https://user-images.githubusercontent.com/114253016/193103755-4fa647f2-0bfb-49d9-9087-d205888e82d6.png)
Foto para
 llamar la atención

**Reply by ByLobs** [reply]
¿Que sistema base Linux estás usando?

**Reply by manuel260300** [reply]
Ubuntu 22.04.1 LTS (Jammy Jellyfish)

**Comment by UncannyChimp** [comment]
Recomiendo que en youtube busques los videos, por que es mejor programar en linux y por que es mejor programar en windows del canal de Hola Mundo, son de buena ayuda para que te des una idea de que ofrece cada una

Y un breve resumen es linux funciona en cualquier sistema operativo pero es un poco mas enrolloso, y windows tiene mas apps ademas de que tambien ya por el hecho de tu comodidad

**Reply by ByLobs** [reply]
Gracias por tu recomendación, vere los videos que comentas

**Comment by Amhedriel** [comment]
En lo personal manejo ambas distribuciones, pero las distribuciones Linux que manejo son algo inestables en mis PC´s puede que sea que són antiguas, más de 6 y 7 años algunos, y los utilizo junto con Windows 10 (por la anitgüedad no es posible actualizar a W11) porque es más estable y no se quedan colgados o se apagan por... no sé, así no pierdo nada.

Ahora, por las integraciones nativas a todo el ecosistema Microsoft (xbox, 360, Outloock, etc.) yo hoy en día prefiero Windows, además, si tuviera PC´s más potentes podría hacer correr Ubuntu de forma nativa sin ningún problema y más estable.

Entiendo que otras personas prefieran otros sistemas operativos IOS por ejemplo que pueden manejar casi la misma manera que Linux, pero lo importante es programar y lo resumiría así:

* Eres nuevo y quieres programar lo más rápido posible con clicks en ventanas de instalación y configuración sin preocuparte mucho de conflictos, y poder ampliar la potencia que pueda tener tu PC, usa **Windows**.
* Eres profesional, y necesitas herramientas acordes a tus necesidades sin importar el costo (o sea ser profesional y ganar como un profesional) usa **IOS**.
* Eres un loco de la computación y no te gusta que te limiten o encierren tu mundo dentro de una *caja bonita* o ver el mundo a través de una *ventana*, lo que quieres ser capaz de manejar el mismo Kernel (recuerda que: "un gran poder conlleva una gran responsabilidad" *tio Ben*) y seguir los pasos de un tal [Linus Torvalds](https://es.wikipedia.org/wiki/Linus_Torvalds "¿Linux?") entonces cualquier distribución **Linux** (ojo con Arch) es lo tuyo.

Abrazos missioner´s.

**Comment by MarioStr1fe** [comment]
Dependiendo de lo que quieras usar y que te acomode mas, en lo personal prefiero mac os que es linux XD o linux, windows solo lo uso para la empresa, por que es su politica.

**Comment by jonathanalfarop** [comment]
Yo creo que todas las personas que  nos dedicamos a las ciencias de la computación deberíamos aprender Linux. Además, saberlo, te abre muchas más oportunidades laborales, si ya vas a aprender desarrollo full stack, que mejor que complementarlo con aprender Linux. Para empezar está bien Ubuntu, aprende a utilizar la terminal y editores como VIM, nano, etc. Verás que en un futuro te sacará de algún apuro.

**Reply by ByLobs** [reply]
Me parece un muy buen punto, lo tomaré en cuenta, gracias!

**Comment by gersonjeanpierre** [comment]
Pues uso Fedora en WSL para desarrollar

**Comment by oramirezj** [comment]
Yo uso particiones de Fedora y Windows en el mismo disco duro, uso ambas, un rato cada una, para practicar en ambos entornos, lo hago desde hace un tiempo, ambas tienen ventajas y desventajas. En desarrollo web, me siento más comodo en linux, pero utilizo ambas, porque segun entiendo en la parte backend de servidores es más comun encontrarte con entornos linux, sin embargo el usar ambas todos los días me permite praticar y aprender de las dos, porque en el uso del día a día es cuando notas las diferencias.
